src/Charcoal/ImageCompression/Service/ImageCompressionService.php

```diff
@@ -254,6 +254,28 @@ private function createObjTable(ModelInterface $proto)
         }
     }
 
+    /**
+     * Returns a case-insensitive pattern useful for the glob function.
+     * For example: "jpg" becomes"[jJ][pP][gG]"
+     *
+     * @param string $pattern The original string to modify.
+     * @return string
+     */
+    private function caseInsensitivePattern(string $pattern) : string
```

Comment:
I suggest mentioning "glob" in the method name for clarity.

```suggestion
    private function caseInsensitiveGlobPattern(string $pattern) : string
```